.openshift-ci/drivers/build.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM builder-base:latest AS builder
```

Comment:
The base image is replaced by Openshift CI. Robby advised me to use something like "FROM will_be_replaced_by_openshift_ci"

Reply:
I honestly don't like having that in a Dockerfile, can we open the discussion up to the rest of the team and see what everyone thinks?

Reply:
I posted about this in slack. Seems like Gavin and Dmitrii like the idea of having a placeholder base. You can keep this as is for now.